src/core/raster/qgscolorrampshader.cpp

```diff
@@ -35,6 +35,7 @@ QgsColorRampShader::QgsColorRampShader( double theMinimumValue, double theMaximu
     , mLUTFactor( 1.0 )
     , mLUTInitialized( false )
     , mClip( false )
+    , mColorRampName( "" )
```

Comment:
Initialization lists like this must follow the same order that the variables are declared in the header. That's what's throwing the build warning and causing Travis to fail here.

But - QStrings shouldn't be initialized (or cleared) with `""`. You can just use the default QString() constructor instead (and clear them with .clear()) . Ie - this line isn't required and can be safely removed.